src/bot.js

```diff
@@ -100,6 +101,9 @@ client.on('message', async msg => {
                 channel: msg.channel,
                 member: member,
                 command: commandString,
+                args: args.filter(el => {
+                    return el != ''
+                }),
```

Comment:
Can probably shorten this to
```suggestion
                args: args.filter(el => el != ''),
```